### PR TITLE
828 - wrong operation group hash fix[backport]

### PR DESCRIPTION
**Pull request by piotrkosecki**
fixing the bug for the release branch

### CODE REVIEW

File: src/main/scala/tech/cryptonomic/conseil/tezos/TezosOptics.scala [hunk 1]
```diff
@@ -105,7 +105,7 @@ object TezosOptics {
       */
     def extractOperationsSequence(
         block: Block
-    ): List[Either[Operation, InternalOperationResults.InternalOperationResult]] =
+    ): List[(Either[Operation, InternalOperationResults.InternalOperationResult], OperationHash)] =
```
Comment (ivanopagano) [inline src/main/scala/tech/cryptonomic/conseil/tezos/TezosOptics.scala:108]: Same consideration as the parallel PR.

Maybe using the opHash as first element in the tuple and possibly using a wrapping case class could improve the reading experience

File: src/main/scala/tech/cryptonomic/conseil/tezos/app/TezosNamesOperations.scala [hunk 2]
```diff
@@ -81,9 +81,9 @@ class TezosNamesOperations(tnsContracts: TNSContract, node: TezosNodeOperator) e
     blocks =>
       blocks.map { b =>
         val refs = extractAppliedTransactions(b).filter {
-          case Left(op) => tnsContracts.isKnownRegistrar(op.destination)
-          case Right(op) => tnsContracts.isKnownRegistrar(op.destination)
-        }.map(tnsContracts.readLookupMapReference).flattenOption
+          case (Left(op), _) => tnsContracts.isKnownRegistrar(op.destination)
```
Comment (ivanopagano) [inline src/main/scala/tech/cryptonomic/conseil/tezos/app/TezosNamesOperations.scala:84]: Here and everywhere else the group hash is not used from code downstream, I would suggest to simply remove it from the list as a first step, to simplify all the code coming after

File: src/main/scala/tech/cryptonomic/conseil/tezos/TezosOptics.scala (hunk 1 above)
Comment (piotrkosecki) [reply src/main/scala/tech/cryptonomic/conseil/tezos/TezosOptics.scala:108]: sure, changed order in the tuple

File: src/main/scala/tech/cryptonomic/conseil/tezos/app/TezosNamesOperations.scala (hunk 2 above)
Comment (piotrkosecki) [reply src/main/scala/tech/cryptonomic/conseil/tezos/app/TezosNamesOperations.scala]: changed

**Pull request merged**